content/libraries/clients.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Client Libraries
+description: Client libraries handle communications between your software and the IOTA network.
+---
+
+# Client Libraries
+
+Client libraries handle communications between your software and the IOTA network. You can use a client library
+to [connect to a node](/iota.rs/examples/get_info)
+, [post a message](/iota.rs/examples/simple_message)
+, [send a signed transaction](/iota.rs/examples/transaction), and many other operations.
+
+![Layered Overview](/img/libraries/layered_overview.svg)
+
+## Iota.rs
+
+The [iota.rs library](/iota.rs/welcome) is written
+in [Rust](/iota.rs/getting_started/rust), and
+has bindings for [Java](/iota.rs/getting_started/java/getting_started)
+, [NodeJs](/iota.rs/getting_started/nodejs)
+, [Python](/iota.rs/getting_started/python)
+and [Wasm](/iota.rs/getting_started/wasm).
+
+Both the IOTA Foundation and the IOTA community maintain the library and this
+wiki, which means that you can rely on it as a source of knowledge on iota.rs.
```

Comment:
Why do we say it about iota.rs but not iota.js?

Also I really like that we are saying out loud who is maintaining the library and its wiki section, but I think we can drop the second part. Not because you can't rely on iota.rs and the docs, but because it's up to the users if they want to rely on something maintained by both IF and community or not.

```suggestion
This library and its documentation are maintained by both IOTA Foundation and the IOTA community.
```

Reply:
cause iota.js, though functionally complete, is still based on iota.rs. So we try and steer people towards .rs